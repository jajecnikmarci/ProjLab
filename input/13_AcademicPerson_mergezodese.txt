starttest
add room room1 3
add item room1 Camembert camembert
add item room1 FFP2 ffp2
add item room1 TVSZ tvsz
add item room1 Transistor transistor
add item room1 Rug rug
add person room1 Student student1
add person room1 Student student2
add person room1 Student student3
pickup student1
pickup student1
pickup student2
pickup student2
pickup student3
printstate student1
printstate student2
printstate student3
use student3 camembert
printstate student1
printstate student2
printstate student3
endtest