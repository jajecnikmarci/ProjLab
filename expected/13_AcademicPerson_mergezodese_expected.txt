student1
	- location: room1
	- stunned: false
	- inventory: 
		rug
		transistor
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student2
	- location: room1
	- stunned: false
	- inventory: 
		tvsz
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		poison1
	- souls: 3
student3
	- location: room1
	- stunned: false
	- inventory: 
		camembert
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student1
	- location: room1
	- stunned: true
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
student2
	- location: room1
	- stunned: false
	- inventory: 
		tvsz
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		poison1
	- souls: 3
student3
	- location: room1
	- stunned: false
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3